app/viewmodels/helpers/ItemDetailsCardHelper.scala

```diff
@@ -0,0 +1,278 @@
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package viewmodels.helpers
+
+import config.AppConfig
+import models.common.UnitOfMeasure.Kilograms
+import models.response.emcsTfe.{MovementItem, Packaging}
+import play.api.i18n.Messages
+import play.twirl.api.Html
+import uk.gov.hmrc.govukfrontend.views.Aliases.{Key, Text}
+import uk.gov.hmrc.govukfrontend.views.viewmodels.content.{Content, HtmlContent}
+import uk.gov.hmrc.govukfrontend.views.viewmodels.summarylist.{SummaryListRow, Value}
+import views.html.components.{link, list}
+
+import javax.inject.Inject
+
+class ItemDetailsCardHelper @Inject()(list: list, link: link, appConfig: AppConfig) {
+
+  def constructItemDetailsCard(item: MovementItem)(implicit messages: Messages): Seq[SummaryListRow] = {
+
+    implicit val _item = item
+
+    Seq(
+      commodityCodeRow(),
+      quantityRow(),
+      grossWeightRow(),
+      netWeightRow(),
+      densityRow(),
+      alcoholicStrengthRow(),
+      maturationAgeRow(),
+      degreePlatoRow(),
+      fiscalMarkRow(),
+      designationOfOriginRow(),
+      sizeOfProducerRow(),
+      brandNameOfProductRow(),
+      commercialDescriptionRow(),
+      wineProductCategoryRow(),
+      wineOperationsRow(),
+      wineGrowingZoneCodeRow(),
+      thirdCountryOfOriginRow(),
+      wineOtherInformationRow()
+    ).flatten
+  }
+
+
+  private def summaryListRowBuilder(key: Content, value: Content) = SummaryListRow(
+    Key(key),
+    Value(value),
+    classes = "govuk-summary-list__row--no-border"
+  )
+
+  private def wineOtherInformationRow()(implicit item: MovementItem, messages: Messages) =
+    item.wineProduct.flatMap(_.otherInformation.map { otherInformation =>
+      summaryListRowBuilder(
+        Text(messages("itemDetails.key.wineOtherInformation")),
+        Text(otherInformation)
+      )
+    })
+
+  private def thirdCountryOfOriginRow()(implicit item: MovementItem, messages: Messages) =
+    item.wineProduct.flatMap(_.thirdCountryOfOrigin.map { country =>
+      summaryListRowBuilder(
+        Text(messages("itemDetails.key.thirdCountryOfOrigin")),
+        Text(country)
+      )
+    })
+
+  private def wineGrowingZoneCodeRow()(implicit item: MovementItem, messages: Messages) =
+    item.wineProduct.flatMap(_.wineGrowingZoneCode.map { zoneCode =>
+      summaryListRowBuilder(
+        Text(messages("itemDetails.key.wineGrowingZoneCode")),
+        Text(zoneCode)
+      )
+    })
+
+  private def wineOperationsRow()(implicit item: MovementItem, messages: Messages) =
+    item.wineProduct.map(_.wineOperations match {
+      case Some(values) if values.nonEmpty =>
+        summaryListRowBuilder(
+          Text(messages("itemDetails.key.wineOperations")),
+          HtmlContent(list(values.map(Html(_))))
+        )
+      case _ =>
```

Comment:
Should this bring back the wine operations description aswell?  currently looks like this 
![image](https://github.com/hmrc/emcs-tfe-frontend/assets/15212062/fd6711cf-d533-4caa-83fc-944fdc4e3bc2)